### PR TITLE
[UP-269] add get license information, update unipdf to v3.30, update LICENSE.md, update README.md

### PR DESCRIPTION
**Pull request by sampila**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,10 @@
...
- * offering services as an application service provider or over-network application programming interface (API)
- * creating/manipulating documents for users in a web/server/cloud application
- * shipping UniDoc with a closed source product
-
-Please see [pricing](http://unidoc.io/pricing) to purchase a commercial license or contact sales at sales@unidoc.io
-for more info.
+The use of this software package is governed by the end-user license agreement
+(EULA) available at: https://unidoc.io/eula/
 
+To obtain a Trial license code to evaluate the software, please visit
```
Comment (gunnsth) [inline LICENSE:9]: Update to get a free metered code

File: LICENSE.md [hunk 2]
```diff
@@ -1,21 +1,10 @@
...
- * offering services as an application service provider or over-network application programming interface (API)
- * creating/manipulating documents for users in a web/server/cloud application
- * shipping UniDoc with a closed source product
-
-Please see [pricing](http://unidoc.io/pricing) to purchase a commercial license or contact sales at sales@unidoc.io
-for more info.
+The use of this software package is governed by the end-user license agreement
+(EULA) available at: https://unidoc.io/eula/
 
+To obtain a Trial license code to evaluate the software, please visit
```
Comment (gunnsth) [inline LICENSE.md:9]: also

**Review by gunnsth — CHANGES_REQUESTED** [review]
Looks great, minor changes in one part

File: LICENSE (hunk 1 above)
Comment (sampila) [reply LICENSE]: updated at 8bd35aeb42ce72eb86b57c7481271c1a95750b86

File: LICENSE.md (hunk 2 above)
Comment (sampila) [reply LICENSE.md]: Updated at 8bd35aeb42ce72eb86b57c7481271c1a95750b86

**Review by gunnsth — APPROVED** [review]
LGTM

**Pull request merged**